### PR TITLE
fix: sort recents by lastAccessedAt

### PR DESCRIPTION
**Pull request by alexnm**
Follow-up of the recent page, sorting the branches and sandboxes merged collection by `lastAccessedAt`, not that the field is present in the graphql query

### CODE REVIEW

**Comment by lbogdan** [comment]
Build for latest commit https://github.com/codesandbox/codesandbox-client/pull/6969/commits/db01ee90bf7fb0534ed858e4c1902fa1b16a7767 is at https://pr6969.build.csb.dev/s/new.

File: packages/app/src/app/graphql/types.ts [hunk 1]
```diff
@@ -1503,12 +1505,15 @@ export type Template = {
   __typename?: 'Template';
   bookmarked: Maybe<Array<Maybe<Bookmarked>>>;
   color: Maybe<Scalars['String']>;
+  /** @deprecated This field is deprecated and will always be null. Query sandbox > description instead */
```
Comment (tristandubbeld) [inline packages/app/src/app/graphql/types.ts:1508]: 👌

**Review by tristandubbeld — APPROVED** [review]
Seems to work for me 👍

**Review by olarclara — APPROVED** [review]
Looks good!

(A bit weird that `build-deps` succeeds but `typecheck` fails)

**Pull request merged**